worker/caasadmission/filter.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package caasadmission
+
+import (
+	apis "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// admissionObjectIgnores defines a slice of GVK's that should be ignored by
+// the caasadmission controller.
+var admissionObjectIgnores = []apis.GroupVersionKind{
+	// ignoring SelfSubjectAccessReview checks becuase of bug lp-1910989
+	{
+		Group:   "authorization.k8s.io",
+		Kind:    "SelfSubjectAccessReview",
+		Version: "v1",
+	},
+}
+
+// compareAPIGroupVersionKind compares two api GroupVersionKind objects for
+// eqauoity.
+func compareAPIGroupVersionKind(a apis.GroupVersionKind, b apis.GroupVersionKind) bool {
+	return a.Group == b.Group && a.Kind == b.Kind && a.Version == b.Version
```

Comment:
Do we need version? Seems overly restrictive
Would group and kind not be sufficient?

Reply:
Yeah version is very important. We could expand this later to support wild carding but I wouldn't leave it out as it's not a complete type without version.